src/Evolution/Systems/ScalarWave/EnergyDensity.hpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace ScalarWave {
+/// @{
+/*!
+ * \brief Computes the energy density of the scalar wave system.
+ *
+ * Below is the fucntion used to calculate the energy density.
+ * \f{align*}
```

Comment:
can you please add a blank line before the `\f{align*}` As in:
```cpp
 * Below is the fucntion used to calculate the energy density.
 * 
 * \f{align*}
```

also typo: `fucntion` -> `function`